cmd/image-builder/manifest.go

```diff
@@ -84,9 +90,10 @@ func generateManifest(dataDir string, extraRepos []string, img *imagefilter.Resu
 		return err
 	}
 	var imgOpts *distro.ImageOptions
-	if opts.Ostree != nil {
+	if opts.Ostree != nil || opts.Subscription != nil {
```

Comment:
Is it OK to have `opts.Ostree` (or `opts.Subscription`) to be `nil`? This is now an `||` so either one could be.

If this is fine then we could drop the check entirely?